cppSettings.gradle

```diff
@@ -72,6 +72,8 @@ task clean(type: Delete) {
     delete buildDir
 }
 
+ext.toolChainPath = null
```

Comment:
Sure it is. Without the null-check, `-PtoolChainPath=<path>` will be ignored. Here's a simple gradle test:

```gradle
ext.test = null

println ext.test

task empty
```

Running `./gradlew empty -Ptest=hello` prints:

```
C:\Users\333fr\test> gradle empty -Ptest=hello
null
:empty UP-TO-DATE

BUILD SUCCESSFUL

Total time: 0.674 secs
```

On the other hand, this script:

```gradle
if (!project.hasProperty('test'))
    ext.test = null

println ext.test

task empty
```

Running `./gradlewtest empty -Ptest=hello` prints:

```
C:\Users\333fr\test> gradle empty -Ptest=hello
hello
:empty UP-TO-DATE

BUILD SUCCESSFUL

Total time: 0.737 secs
```